### DISCUSSION TITLE
[Bug]: XDebug Herd Pro not breaking

### DISCUSSION THREAD

**Discussion by JaxonDupre** [beyondcode/herd-community#1029] [General]
### Platform

macOS

### Operating system version

macOS Darwin 13.5.1

### System architecture

Intel (x86)

### Herd Version

1.9.1

### PHP Version

PHP 7.4

### Bug description

Laravel Herd Pro with XDebug should break on file breakpoint; Instead it breaks on Herd dump-loader.php and returns my connection request information to my local machine's herd server including cache, GET requests, and the cookies including my PHPSTORM token. If I remove force-break when a file is outside the project I many errors debug session finished without pausing.

### Steps to reproduce

1. Add local server to PHPStorm
<img width="741" alt="Screenshot 2024-08-16 at 10 10 22 PM" src="https://github.com/user-attachments/assets/6f7a4045-7f84-461d-9b6b-b4d38c909de8">
2. Configure PHPStorm
<img width="739" alt="Screenshot 2024-08-16 at 10 11 52 PM" src="https://github.com/user-attachments/assets/91ba1365-7a8a-4a4d-a48e-b60014c1cf16">
3. Add breakpoint to file, open connections, and refresh site
4. Met with connection from dump-loader.php
<img width="1335" alt="Screenshot 2024-08-16 at 10 27 04 PM" src="https://github.com/user-attachments/assets/6e687521-5751-470e-84ce-4bd1ad620b02">

### Other

If there is no server defined in PHPStorm and "break at first PHP" is enabled then I receive a prompt to add my server.test:80 with the dump-loader.php being the file that is caught once again which includes all of my GET informations and cookies and whatnot.

**Comment by mpociot** [comment]
I don't fully understand the issue you're mentioning.
What do you mean by " If I remove force-break when a file is outside the project I many errors debug session finished without pausing."

I also configured PHPStorm the way you have it configured and I can hit all breakpoints and the application is pausing.

As the `dump-loader.php` file is the entry point (because of the auto_prepend_file directive) there is no way to avoid this behavior.
